controller/tests/controller_server/csi_controller_server_test.py

```diff
@@ -1594,43 +1594,48 @@ def test_expand_volume_with_no_space_in_pool_exception(self):
 
 class TestIdentityServer(BaseControllerSetUp):
 
-    @patch.object(CSIControllerServicer, "get_identity_config")
+    @patch("controller.controller_server.common_config.config.identity")
     def test_identity_plugin_get_info_succeeds(self, identity_config):
         plugin_name = "plugin-name"
         version = "1.1.0"
-        identity_config.side_effect = [plugin_name, version]
+        identity_config.name = plugin_name
+        identity_config.version = version
         request = Mock()
         context = Mock()
         request.volume_capabilities = []
         response = self.servicer.GetPluginInfo(request, context)
         self.assertEqual(response, csi_pb2.GetPluginInfoResponse(name=plugin_name, vendor_version=version))
 
-    @patch.object(CSIControllerServicer, "get_identity_config")
+    @patch("controller.controller_server.common_config.config.identity")
     def test_identity_plugin_get_info_fails_when_attributes_from_config_are_missing(self, identity_config):
         request = Mock()
         context = Mock()
 
-        identity_config.side_effect = ["name", Exception(), Exception(), "1.1.0"]
-
+        identity_config.name = "name"
+        identity_config.version = Exception()

```

Comment:
this causes a `TypeError` [here](https://github.com/IBM/ibm-block-csi-driver/blob/b43bc0ee5e334bf34a4fc6764710185851b22775/controller/controller_server/csi_controller_server.py#L466), whereas before this change, the second item in the (removed) list would cause an `Exception` [here](https://github.com/IBM/ibm-block-csi-driver/blob/857c8ebb333206698cca5eb3a07fdecfa0767fcf/controller/controller_server/csi_controller_server.py#L475), as desired